packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.styles.ts

```diff
@@ -1,28 +1,75 @@
 import { IContextualMenuStyleProps, IContextualMenuStyles } from './ContextualMenu.types';
-// import {
-//   IStyle,
-//   ITheme,
-// } from '../../Styling';
+import { getGlobalClassNames, FontWeights } from '../../Styling';
+
+const GlobalClassNames = {
+  root: 'ms-ContextualMenu is-open',
```

Comment:
This should be pulled into two separate classes - see that you're also doing the same for `list` for `is-open`.